generators/ci-cd/templates/.gitlab-ci.yml.ejs

```diff
@@ -162,12 +162,13 @@ maven-compile:
 maven-test:
     stage: test
```

Comment:
Minor issue, but for gradle I have added a new step `integration-test` such that they are executed in different jobs, but we can leave it like this.

Reply:
@atomfrede I have initially this step also but since integration-test maven phase already include test phase, it's not make sence to separate them (https://github.com/jhipster/generator-jhipster/pull/9337/commits/8987f8159c4723fb27ea013cd08d6766ea15759c).

Also for consistency with the other ci implementation, we have only 1 step for backend tests...

Reply:
:heavy_check_mark: Your are right, in gradle that is not the case.